### DISCUSSION TITLE
part-ii/2020-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#237] [Question comments]
Part II 2020 Logic and Set Theory: Paper 3, Section II, $16 \mathrm{H}$ 

https://questions.tripos.org/part-ii/2020-88/

**Comment by artie2000** [comment]
Most of this question is fairly routine checking. I found the following formula useful:

$$\text{TC}(\{x\})=\{x\}\cup\bigcup_{y\in x}\text{TC}(\{y\}).$$

<details>
<summary>Last bit:</summary>

The Axiom of Union fails: consider the cardinality of $\bigcup\{y\}$.

</details>
